.github/workflows/carpenter_build.yaml

```diff
@@ -2,25 +2,42 @@ name: Carpentry
 on:
   pull_request:
 env:
-  IMAGE_NAME: arcaflow-plugin-kubeconfig
+  IMAGE_NAME: arcaflow-plugin-template-python
```

Comment:
Oops -- Need to update the name.

Reply:
darn you are right. not the first time I have done that mistake from copy pasta. Thank you